src/client/datascience/interactive-ipynb/nativeEditorCommandListener.ts

```diff
@@ -34,7 +34,9 @@ export class NativeEditorCommandListener implements IDataScienceCommandListener
             commandManager.registerCommand(Commands.NotebookEditorRemoveAllCells, () => this.removeAllCells())
         );
         this.disposableRegistry.push(
-            commandManager.registerCommand(Commands.NotebookEditorInterruptKernel, () => this.interruptKernel())
+            commandManager.registerCommand(Commands.NotebookEditorInterruptKernel, (document: NotebookDocument) =>
```

Comment:
When you run this from the command palette doesn't it pass the uri of the file as the first argument? At least execSelectionInInteractive window did that (as in the command palette always passes the URI of the active editor)

Reply:
It doesn't seem to pass anything when invoked from the command palette:

![image](https://user-images.githubusercontent.com/30305945/113903443-db435680-9785-11eb-9b80-2e8a45536f08.png)

Reply:
Might depend upon what has focus when the palette comes up?

Reply:
Yeah, I was at least semi-sure that it was the active editor document. It's annoying to track down what parameters are passed for these I've found. I spelunked around in VS Code source before, but I've never found it explicit in the API anywhere that I looked.

Reply:
I just tried invoking execSelectionInteractive from the command palette with a selection in a Python file, and I don't see anything being passed as an arg to the handler 😕  I do see the URI being passed when invoked from the context menu, but that's a different scenario. Ian, do you remember how you got that to happen for you?

![image](https://user-images.githubusercontent.com/30305945/113910217-6d029200-978d-11eb-95d8-5d3636d2b9d2.png)


Similarly after setting focus to the notebook cell or the notebook editor, I'm still not seeing the URI getting passed. However at minimum the parameter can clearly be undefined so I should update the type in the handler.